.github/workflows/wheel_manylinux.yml

```
@@ -0,0 +1,67 @@
+name: wheel - manylinux2014
+
+on:
+  push:
+    tags:
+      - '*'
+jobs:
+  wheel_manylinux:
+    name: manylinux2014
+    runs-on: ${{ matrix.os }}
+    container:
+      image: quay.io/pypa/manylinux2014_x86_64:latest
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        python-version: [cp37-cp37m, cp38-cp38]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        run: |
+          echo "::add-path::/opt/python/${{ matrix.python-version }}/bin/"
+      - name: check C++ compiler and Python
+        run: |
+          gcc --version
+          python --version
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade --no-cache-dir pip
+          python -m pip install --no-cache-dir "poetry<2,>=1.0"
+          python -m pip install --no-cache-dir "mypy-protobuf~=1.0"
+      - name: Build libraries
+        run: |
+          make
+      - name: Build wheel
+        run: |
+          python -m poetry build -v
+      - name: Fix wheel
+        env:
+          PLAT: manylinux2014_x86_64
+        run: |
+          auditwheel repair dist/*.whl --plat "${PLAT}" -w "./wheelhouse"
```

```suggestion
          auditwheel repair dist/*.whl -w wheelhouse
```
I'm still a bit confused here. Is the wheelhouse directory already created? If not we need to create it.

---

Sorry, I see that -w takes a directory as an argument. It creates it if it doesn't exist I gather?

---

Yes